### PR TITLE
grub-reboot: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

It resolves part of issue #8080

### CODE REVIEW

File: pages/linux/grub-reboot.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# grub-reboot
+
+> Set the default boot entry for GRUB, for the next boot only.
+> More information: <https://manned.org/grub-reboot>.
+
+- Set the boot entry to a entry number, name or identifier:
+
+`sudo grub-reboot {{menu_entry}}`
```
Comment (blueskyson) [inline pages/linux/grub-reboot.md:8]:
```suggestion
`sudo grub-reboot {{entry_number}}`
```

File: pages/linux/grub-reboot.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Set the boot entry to a entry number, name or identifier for alternative boot directory:
+
+`sudo grub-reboot --boot-directory {{/path/to/directory}} {{menu_entry}}`
```
Comment (blueskyson) [inline pages/linux/grub-reboot.md:12]:
```suggestion
`sudo grub-reboot --boot-directory {{/path/to/boot_directory}} {{entry_number}}`
```

File: pages/linux/grub-reboot.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# grub-reboot
+
+> Set the default boot entry for GRUB, for the next boot only.
+> More information: <https://manned.org/grub-reboot>.
+
+- Set the boot entry to a entry number, name or identifier:
```
Comment (kbdharun) [inline pages/linux/grub-reboot.md:6]:
```suggestion
- Set the default boot entry to an entry number, name or identifier for the next boot:
```

File: pages/linux/grub-reboot.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`sudo grub-reboot {{menu_entry}}`
+
+- Set the boot entry to a entry number, name or identifier for alternative boot directory:
```
Comment (kbdharun) [inline pages/linux/grub-reboot.md:10]:
```suggestion
- Set the default boot entry to an entry number, name or identifier for an alternative boot directory for the next boot:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**